content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -163,16 +163,21 @@ There are a few limitations with `env` at the moment:
 
 For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files. The Prisma CLI expects to find the `.env` file:
 
-1. Next to your schema file
-2. In the `./prisma` folder, if it is not found next to your schema file
+1. In the root folder of your project (`./.env`)
+1. From the location specified by the `--schema` argument
+1. From the location of the schema taken from `"prisma": {"schema": "/path/to/schema.prisma"}` in `packge.json`
+1. From the `./prisma` folder
+
+If a `.env` file is located in step #1, but additional, clashing `.env` files are located in steps #2 - 4, the CLI will throw an error.
```

Comment:
What does that error look like?

Reply:
https://github.com/prisma/prisma/blob/master/src/packages/cli/src/__tests__/__snapshots__/dotenv-3-conflict.test.ts.snap#L14